### DISCUSSION TITLE
nonebot连接LLOneBot中使用问题，nb命令行窗口一定时间以后卡住不刷消息

### DISCUSSION THREAD

**Discussion by 3CB3EC** [nonebot/discussions#476] [问答]
想请教下各位大佬，遇到这么个问题：

机器人启动正常，开始能够正常运行。一段时间后nonebot命令行窗口不再刷新消息，给机器人发送消息也无任何反应。此时nonebot命令行窗口无任何报错。
在nonebot命令行窗口中按一下回车，这时刚刚卡住的消息全部依次刷新出来，机器人也会根据其中消息正确做出对应的反应，中间没有输出任何ERROR。

现在不太确定是nb的问题还是协议端的问题所以想现在这边问问没有去提issue，希望大佬指点一下

=======
部署环境
版本	Windows Server 2022 Standard
版本号	21H2
安装日期	‎2023-‎06-‎20
操作系统内部版本	20348.1787

Python版本3.10.11
nb版本2.3.1
适配器OneBot V11 2.4.4
协议端LLOneBot v3.26.7

**Comment by yanyongyu** [comment]
这是你终端选中了吧。。。关闭终端的快速选择模式就可以了

**Reply by 3CB3EC** [reply]
> 这是你终端选中了吧。。。关闭终端的快速选择模式就可以了

是这样的，我看消息正常刷，我就切出去做别的事去了，背景cmd窗口看到消息还在正常刷的，过一段时间回来看就卡住不动了

**Reply by yanyongyu** [reply]
你把快速选择模式关了再看看卡不卡就知道了，多半是你误点了终端窗口

**Reply by 3CB3EC** [reply]
> 你把快速选择模式关了再看看卡不卡就知道了，多半是你误点了终端窗口

谢谢，我去试试

**Reply by 3CB3EC** [reply]
> 你把快速选择模式关了再看看卡不卡就知道了，多半是你误点了终端窗口

好像确实有用！一晚上没出现那个问题，谢谢！
